models/llm_openai/model.py

```
@@ -0,0 +1,76 @@
+        in his city.
+
+        - If you judge that the post describes, reports, or implies moral outrage, respond with: "1"
+        - If the post is unrelated or does not describe moral outrage, respond with: "0"
+
+        Only output your label. ONLY output 0 or 1.
+
+        Post: {text}
+        Answer:
+        """
+
+        input = [
```

Ruff (0.15.9)

[error] Variable `input` is shadowing a Python builtin

(A001)